CONTRIBUTING.rst

```diff
@@ -36,6 +39,47 @@ If modifying dependencies in ``mlflow/server/js/package.json``, run ``npm update
 ``mlflow/server/js`` to install the updated dependencies.
 
 
+Java API
+--------
+
+The MLflow Java client depends on the Python client, so first install the Python client in a conda
+environment as described above. `Install <https://www.oracle.com/technetwork/java/javase/downloads/index.html>`_
+the Java 8 JDK (or above), and `download <https://maven.apache.org/download.cgi>`_
+and `install <https://maven.apache.org/install.html>`_ Maven. You can then build and run tests via:
+
+.. code:: bash
+
+  cd mlflow/java
+  mvn clean package
+
+R API
+-----
+
+The MLflow R client depends on the Python client, so first install the Python client in a conda
+environment as described above. Install R, then run the following to install dependencies for
```

Comment:
Could we link to installing R? Like how to get Rscript.

Reply:
Btw, it seems like conda works fine here -- i.e., `conda install r`. We should probably advise that.

Reply:
Or maybe not, I get

```
NOT_CRAN=true Rscript -e 'install.packages("devtools", "https://cloud.r-project.org")'
dyld: Library not loaded: @rpath/libintl.9.dylib
```

Were you able to get this running locally? What was your setup?

Reply:
Good question, I had an existing R installation on my laptop (not sure from when/where). I'll retry these steps with a clean installation from https://cloud.r-project.org/